docs/en/wallet-instance-attestation.rst

```diff
@@ -43,221 +43,41 @@ The following requirements are assumed for the Wallet Instance Attestation:
-+------------------------------------+------------------------------------+
-|| jwks                              || Containing the keys attribute     |
-||                                   || which is an array of all the      |
-||                                   || Wallet Provider's public keys.    |
-+------------------------------------+------------------------------------+
-|| token_endpoint                    || Endpoint for obtaining the Wallet |
-||                                   || Instance Attestation.             |
-+------------------------------------+------------------------------------+
-|| asc_values_supported              || List of supported values for      |
-||                                   || the certified security context.   |
-||                                   || These values define a level of    |
-||                                   || assurance about the security of   |
-||                                   || the app. In particular we will    |
-||                                   || mainly have 3 values associated   |
-||                                   || with low, medium and high         |
-||                                   || security. An attested security    |
-||                                   || context is defined according to   |
-||                                   || the proof that the Wallet         |
-||                                   || Instance is able to send to the   |
-||                                   || Wallet Provider.                  |
-||                                   || ⚠️ This parameter is not standard |
-||                                   || and is still under discussion.    |
-+------------------------------------+------------------------------------+
-|| grant_types_supported             || The type of grants supported by   |
-||                                   || the endpoint token. Therefore,    |
-||                                   || for the Wallet Provider the token |
-||                                   || is equivalent only to the Wallet  |
-||                                   || Instance attestation, therefore   |
-||                                   || this attribute will contain an    |
-||                                   || array with only one element.      |
-+------------------------------------+------------------------------------+
-|| token_endpoint_auth_methods_suppo || Supported authentication method   |
-|| rted                              || for the endpoint token.           |
-||                                   ||                                   |
-+------------------------------------+------------------------------------+
-|| token_endpoint_auth_signing_alg_v || List of supported signature       |
-|| alues_supported                   || algorithms.                       |
-+------------------------------------+------------------------------------+
```

Comment:
Sorry @peppelinux, but due to the formatting it is difficult to understand what has been modified. Can we find a solution to this problem?

Reply:
the Wallet Provider Entity Configuration was moved into the wallet solution section as mentioned in the PR description, so it is simple removed from here